### DISCUSSION TITLE
search node unstability ...urgent!

### DISCUSSION THREAD

**Discussion by witekenea** [Security-Onion-Solutions/securityonion#12730] [Unsupported Versions]
Hello,  I have installed the security onion distributed version with one manager node, one sensor node and on search node. The installation process of each node was smooth and successful.First i add a sensor node to the manager node then the search node. the sensor node and the search node are shown in the grid and all services are running...the thing is all of them were shutdown cause of a poweroutrage and when i try to start each only one of the two works...like if the manager node and the sensor node is up the search node will shutdown itself and when i try to start the search node when the manager node and the sensor node is still up the manager node will shutdown itself and now the sensor node and the search node is up. How can i fix this?

**Comment by samanosuke26** [comment]
Are all 3 of your nodes virtualized? And if so, are they virtualized on the same hypervisor?

**Comment by witekenea** [comment]
this installation is on proxmox and each vm (the manager node, search node and sensor node) is under the same node
